buildscript/src/main/kotlin/Dependencies.kt

```diff
@@ -24,7 +24,7 @@ object Dependencies {
     val retrofitConverterGson = "com.squareup.retrofit2:converter-gson:${Versions.retrofit}"
     val okhttp = "com.squareup.okhttp3:okhttp:${Versions.okhttp}"
     val okhttpLogging = "com.squareup.okhttp3:logging-interceptor:${Versions.okhttp}"
-    val okio = "com.squareup.okio:okio:2.1.0"
+    val okio = "com.squareup.okio:okio:2.7.0"
```

Comment:
Why do we use a custom version? It overrides a tested one from okhttp.

Reply:
Good question; 
my logic here that: 
1. we already using 2.6.0 in avito project
2. 2.x is api compatible and mostly consist of performance improvements and bugfixes

Reply:
OkHttp 4.8.0 already uses Okio 2.7.0.
I'd preferer to override transitive dependencies only when we have reasons.